playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_positioning.html.erb

```diff
@@ -0,0 +1,10 @@
+<br />
+<%= pb_rails("title", props: { text: title, tag: "h4", size: 4 }) %>
+
+<% if defined?(usage) && usage.present? %> 
+    <%= pb_rails("body", props: {"tag": "p"}) %>
+   <% usage %>
+<% end %>
+    
+<br />
+
```

Comment:
Let's remove any `<br />` tags and use our spacing props instead (ex: `margin_bottom: "md"`)